client/.env

```diff
@@ -0,0 +1 @@
+VUE_APP_PATH = /nameRequest
```

Comment:
As a result of this, please update the `publicPath` property in vue.config.js (same as in Filings UI).

Reply:
I was curious about this. I thought about it but hesitated. 
If we do that, do we need to set this variable in each OS env?

Reply:
The Vue config file should pick it up from the environment.

This project doesn't have a `.env.production` file. It might be needed. Dunno.

Reply:
Thought about that as well. Prod would essentially be the same, thought not entirely sure if it's required. Will look into it.

Reply:
So after looking into this, the contents of the `.env` file will be accessible in all environments. Since the Path is consistent between local and deployments, we should be good with just a `.env`.